src/datachain/toolkit/diff.py

```diff
@@ -0,0 +1,111 @@
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, Optional, Union
+
+from datachain.lib.diff import CompareStatus, get_status_col_name
+from datachain.lib.diff import compare as chain_compare
+from datachain.query.schema import Column
+
+if TYPE_CHECKING:
+    from datachain.lib.dc import DataChain
+
+
+C = Column
+
+
+def compare(
+    left: "DataChain",
+    right: "DataChain",
+    on: Union[str, Sequence[str]],
+    right_on: Optional[Union[str, Sequence[str]]] = None,
+    compare: Optional[Union[str, Sequence[str]]] = None,
+    right_compare: Optional[Union[str, Sequence[str]]] = None,
+    added: bool = True,
+    deleted: bool = True,
+    modified: bool = True,
+    unchanged: bool = False,
```

Comment:
Do we need these arguments? Can we leave this up to the user to filter? 
```python
dc.compare(...).filter(C("col") == "added")
```

Reply:
`DataChain.compare()`  returns new chain with that column which can be filtered and this new toolkit method does exactly what you described, so it saves the user that one filter step. Now, we can discuss if toolkit method is even needed in the first place.